charts/tigera-operator/templates/_helpers.tpl

```diff
@@ -5,3 +5,16 @@
 {{- end -}}
 {{- .image -}}:{{- .version -}}
 {{- end -}}
+
+{{/*
+generate imagePullSecrets for installation and deployments
+by combining installation.imagePullSecrets with toplevel imagePullSecrets.
+*/}}
+
+{{- define "tigera-operator.imagePullSecrets" -}}
+{{- $secrets := default list .Values.installation.imagePullSecrets -}}
```

Comment:
I suppose if a secret is defined in both places we'll get an error back from k8s? 

This is probably OK, although I can definitely imagine users trying to put a secret in both places.